src/kapla/projects/pyproject.py

```diff
@@ -340,12 +341,13 @@ async def poetry_publish(
 
     @classmethod
     def find_current(
-        cls: Type[PyProjectT], start: Union[None, str, Path] = None
+        cls: Type[PyProjectT], start: Union[None, str, Path] = None, venv_path: str = None
     ) -> PyProjectT:
         """Find project from current directory by default"""
         projectfile = lookup_file("pyproject.toml", start=start)
         if projectfile:
-            return cls(projectfile)
+            venv_path = os.environ["VIRTUAL_ENV"]
```

Comment:
Can we do `os.environ.get("VIRTUAL_ENV", ".venv")` in case the variable is not defined ?

Reply:
Maybe we should do `os.environ.get("VIRTUAL_ENV", None)`. Keeping the information that this is the default value is usefull here because we later set it to `self.root / ".venv"` with the `self.root` prefix in `kapla.projects.base.BasePythonProject __init__ method

Reply:
OK pour ca effectivement il faut qu'on ait none